mailpoet/tests/performance/config.js

```diff
@@ -17,7 +17,7 @@ export const firstName = 'John';
 export const lastName = 'Doe';
 export const defaultListName = 'Newsletter mailing list';
 
-export const thinkTimeMin = '1';
+export const thinkTimeMin = '2';
```

Comment:
Yes, this is added on purpose. I'd like to use it in tests instead of sleep(2), as it is [added here](https://github.com/mailpoet/mailpoet/pull/5879/files#diff-3d8a23534fcadfd2f81497f14f747ce497a217b59e13d638b08d9f7df0d5590bL75). K6 recommends placing sleeps in tests to slow down loading the pages to fully test the performance. Anyway I keep it very low and rarely use sleeps in tests but instead wait for other elements to load or the whole page.

To use `await` (and async somewhere in order to place `await`) is the new normal in k6 and is [required by the migration guide here](https://grafana.com/docs/k6/latest/using-k6-browser/migrating-to-k6-v0-52/).

Thanks for the review!

Reply:
@veljkho Thank you for providing the details and the link to the migration guide. Seeing the migration guide helped me to understand why the changes were done 👍